frontend/src/container/PipelinePage/Layouts/ChangeHistory/index.tsx

```diff
@@ -1,24 +1,48 @@
 import { Table } from 'antd';
+import { useEffect } from 'react';
 import { Pipeline } from 'types/api/pipeline/def';
 
 import { changeHistoryColumns } from '../../PipelineListsView/config';
 import { HistoryTableWrapper } from '../../styles';
 import { historyPagination } from '../config';
 
-function ChangeHistory({ piplineData }: ChangeHistoryProps): JSX.Element {
+function ChangeHistory({
+	refetchPipelineLists,
+	pipelineData,
+}: ChangeHistoryProps): JSX.Element {
+	useEffect(() => {
```

Comment:
would recommend to avoid use of useEffect and make use of `refetchInterval` from useQuery

Reply:
i guess we can write custom logic in the refetchInterval also